apps/admin_app/lib/admin_app_web/views/layout_view.ex

```diff
@@ -36,7 +37,8 @@ defmodule AdminAppWeb.LayoutView do
   end
 
   def check_general_settings() do
-    Repo.all(GeneralConfiguration) |> List.first()
+    count = Repo.aggregate(from(g in "snitch_general_configurations"), :count, :id)
+    if count == 1, do: true, else: false
```

Comment:
This method is too complex to identify the presence of GC or not, please simplify this.

Reply:
```elixir
def check_general_settings do
    case Repo.one(GC)
      nil -> false
      _ -> true
    end
  end
```

Reply:
yeah sure